react/src/components/datavis/DataTable/DataTable.md

```diff
@@ -0,0 +1,2 @@
+### Description
```

Comment:
can we rename this folder in the components from `datavis` to `dataviz` to match storybook syntax.